jira/client.py

```diff
@@ -1499,13 +1499,18 @@ def create_issue(
         p = data["fields"]["project"]
 
         if isinstance(p, (str, int)):
-            data["fields"]["project"] = {"id": self.project(str(p)).id}
+            project_id = self.project(str(p)).id
+            data["fields"]["project"] = {"id": project_id}
+        else:
+            project_id = None
```

Comment:
preference would be for this to be before the if statement. GitHub is a bit silly and doesn't let me suggest something as a "line is deleted" 

But it would look like:

```python
project_id = None
if condition:
    project_id = new_id
```